stdlib/mimetypes.pyi

```diff
@@ -44,7 +44,7 @@ class MimeTypes:
     encodings_map: dict[str, str]
     types_map: tuple[dict[str, str], dict[str, str]]
     types_map_inv: tuple[dict[str, str], dict[str, str]]
-    def __init__(self, filenames: tuple[str, ...] = (), strict: bool = True) -> None: ...
+    def __init__(self, filenames: tuple[StrPath, ...] = (), strict: bool = True) -> None: ...
```

Comment:
It seems it just works.

As a side note -- is there a particular reason why `init` accepts a `Sequence` of names, but `MimeTypes.__init__` only a tuple? I mean other than it being initialised that way. It's working with either tuple or list.